applications/matter_bridge/Kconfig.sysbuild

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF_DEFAULT_BLUETOOTH
+	default y if ($(BOARD) = "nrf7002dk_nrf5340_cpuapp")
+
+choice BOOTLOADER
+	default BOOTLOADER_MCUBOOT
+endchoice
+
+if BOOTLOADER_MCUBOOT && ($(BOARD) = "nrf7002dk_nrf5340_cpuapp")
+
+config MCUBOOT_UPDATEABLE_IMAGES
+	default 2
+
```

Comment:
The problem I have with things like this, is it's not so much setting a default for an application for e.g. a bootloader, it's setting up a tonne of configuration for an image, which I think would be best done in a .conf file that sysbuild uses - thoughts @tejlmand ? This will set the application up in a certain way if the user meets the criteria, but if they for example disable mcuboot in the main configuration command, then enable it in `menuconfig`, none of this configuration will be applied.

Reply:
Ok, actually it's not a problem for me to move it to the `sysbuild.conf`. I added it to Kconfig, as a defaults, because we used to do that in child-image pattern. The reason was that we had multiple `prj_x.confs`, so it was easier to set it in one place instead of copying multiple times across different files.

Reply:
@nordicjm the only problem is that we use different SB configs for different boards, so we cannot apply single `sysbuild.conf` to the multiple targets.

Reply:
`-DSB_CONF_FILE=with_dfu.conf` ?

Reply:
This file is maybe not the perfect example, because Matter bridge is supported only for one board - nRF7002 DK. However the other examples are supported for nRF52840, nRF5340 and nRF7002. Following the `-SB_CONF_FILE` approach we would need to create separate conf files for them and instruct the user how to attach them depending on used board. Unless, is it possible to create in `boards` directory for sysbuild and add `nrf52840dk_nrf52840.conf`, `nrf5340dk_nrf5340_cpuapp.conf` and `nrf7002dk_nrf5340_cpuapp.conf` there, so it would be automatically merged with all others SB configs? We had it for child images, but I'm not sure if it's even possible with sysbuild